src/lib/Form/Processor/Content/AutosaveProcessor.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzPlatformAdminUi\Form\Processor\Content;
+
+use eZ\Publish\API\Repository\Exceptions\Exception as APIException;
+use EzSystems\EzPlatformAdminUi\Event\AutosaveEvents;
+use EzSystems\EzPlatformContentForms\Event\FormActionEvent;
+use EzSystems\EzPlatformContentForms\Form\Processor\ContentFormProcessor;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpFoundation\Response;
+
+class AutosaveProcessor implements EventSubscriberInterface
+{
+    /** @var \EzSystems\EzPlatformContentForms\Form\Processor\ContentFormProcessor */
+    private $innerContentFormProcessor;
+
+    public function __construct(
+        ContentFormProcessor $innerContentFormProcessor
+    ) {
+        $this->innerContentFormProcessor = $innerContentFormProcessor;
+    }
+
+    public static function getSubscribedEvents(): array
+    {
+        return [
+            AutosaveEvents::CONTENT_AUTOSAVE => ['processAutosave', 10],
```

Comment:
It seems that `AutosaveEvents::CONTENT_AUTOSAVE`is never dispatched so whole  `AutosaveProcessor` is a dead code.

Reply:
I can confirm that. I think it should be triggered because it provides a lot of value if you need to intercept autosaving and run your custom code.

Reply:
Dispatched by Content Forms ActionDispatcher: https://github.com/ezsystems/ezplatform-content-forms/blob/master/src/lib/Form/ActionDispatcher/AbstractActionDispatcher.php#L50